watson/cli.py

```diff
@@ -1006,6 +1015,15 @@ def log(watson, current, reverse, from_, to, projects, tags, year, month, week,
     if from_ > to:
         raise click.ClickException("'from' must be anterior to 'to'")
 
+    if bool(projects and ignore_projects and
+            set(projects).intersection(set(ignore_projects))):
+        raise click.ClickException(
+            "given projects can't be ignored at the same time")
+
+    if bool(tags and ignore_tags and set(tags).intersection(set(ignore_tags))):
+        raise click.ClickException(
+            "given tags can't be ignored at the same time")
+
```

Comment:
Frames filtering to ignore projects and tags is missing here (see above filtering and the [frames filtering method](https://github.com/TailorDev/Watson/blob/master/watson/frames.py#L149-L182)).

Reply:
my fault, i cut the diff of my previous PR on the wrong line and the change of the filter itself was not applied to
```
    filtered_frames = watson.frames.filter(
        projects=projects or None, tags=tags or None, span=span
    )
```

Next commit should fix that

Reply:
@jmaupetit you should now see the change below